tests/system/action/motion/test_create_forwarded.py

```diff
@@ -28,6 +30,7 @@ def setUp(self) -> None:
             "motion_state/34": {
                 "name": "name_state34",
                 "meeting_id": 2,
+                "allow_motion_forwarding": True,
```

Comment:
The test data isn't correct. motion_state/34 is a state of the target-meeting, but is also used in the source motion/12 of meeting/1 as current state.
All this data is valid only in their own meeting and cannot be used in the other meeting, too.
The target-motion should use the default workflow of the new meeting with the first state of it.
Please check, if there are more fields form source-motion used in target-motion, i.e. amendments, attachments. The may not be copied to the target motion.
Use tests to show the behavior in this cases.

Reply:
Added a state 30 of the source meeting.
Check the state_id of the created motion.
The create_forwarded action is a **create action**, so no coping of amendmens, etc. No need to test that.